src/theme/typography.js

```diff
@@ -0,0 +1,107 @@
+// override mui default
+// https://material-ui.com/customization/default-theme/
+
+const typography = {
+  htmlFontSize: 16,
```

Comment:
What does this do?

Reply:
i think it sets the font-size in the html tag to 16px, but unsure. I am just using the template provided by the default theme
https://material-ui.com/customization/default-theme/